db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -5,7 +5,7 @@ def up
     Host::Managed.reset_column_information
     Hostgroup.reset_column_information
 
-    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
+    Host::Managed.where(:type => 'Host::Managed').includes(:primary_interface, :hostgroup, :interfaces, :domain, :operatingsystem).find_in_batches(:batch_size => 1000) do |group|
```

Comment:
Why are these fields needed / retrieved at all? I've tried to check the set of instructions happening upon a lookup_value_match= call and it seems to be simply to retrieve the name or get it from the fqdn fact